package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "twitch-hdt-frontend",
-	"version": "1.0.1",
+	"version": "0.5.0",
```

```suggestion
	"version": "1.0.1",
```
This needs to be removed from the PR, we'll do that separately after this is merged.